config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
We will still write in Brazillian Portuguese.

The two languages are 95%+ the same.